src/functional/ffront/field_operator_ast.py

```diff
@@ -127,9 +127,12 @@ class BinOp(Expr):
 
 
 class CompareOperator(StrEnum):
-    GT = "greater"
-    LT = "less"
     EQ = "eq"
+    NOTEQ = "not_equal"
```

Comment:
Either this
```suggestion
    EQ = "eq"
    NOTEQ = "not_eq"
```
To keep it consistent with EQ.